frost-core/src/frost.rs

```diff
@@ -223,13 +232,18 @@ where
 
     /// Compute the preimages to H3 to compute the per-signer binding factors
```

Comment:
s/H3/H1/ ?

https://github.com/ZcashFoundation/frost/blob/a02649d75685c20d972d15ff47ff04084d03fd8a/frost-core/src/frost.rs#L123

```suggestion
    /// Compute the preimages to H1 to compute the per-signer binding factors
```